ddtrace/propagation/http.py

```diff
@@ -710,6 +710,9 @@ def _get_sampling_priority(traceparent_sampled, tracestate_sampling_priority):
     def _extract(headers):
         # type: (Dict[str, str]) -> Optional[Context]
 
+        if "http_x_cloud_trace_context" in headers:
```

Comment:
Google sets `x-cloud-trace-context` and `traceparent` headers when our cloud function is triggered, used for Google Cloud Tracing. Because of the is, the tracer sets the `parent_id` of our cloud function trace's root span, thereby causing the span to not show up under the `Trace Root Spans` in Datadog.

This change prevents the traceparent context from being used to set the span context, so cloud function trace root spans show up properly as trace root spans.